recipes-core/packagegroups/packagegroup-ni-ptest-smoke.bb

```diff
@@ -36,15 +35,10 @@ RDEPENDS:${PN}:append = "\
 	pango-ptest \
 	parted-ptest \
 	perl-ptest \
-	pstore-save-ptest \
```

Comment:
This is one of our recipes.

https://github.com/ni/meta-nilrt/tree/nilrt/master/kirkstone/recipes-ni/pstore-save

Can we can an eval of why the ptest doesn't work?

Reply:
@amstewart This is the output of the pstore-save
```
START: ptest-runnerThe ptest output failed to report any results.
Output:
2024-10-03T17:34The ptest output failed to report any results.
Output:
BEGIN: /usr/lib/pstore-save/ptestThe ptest output failed to report any results.
Output:
The ptest output failed to report any results.
Output:
ERROR: Exit status is 1The ptest output failed to report any results.
Output:
DURATION: 0The ptest output failed to report any results.
Output:
END: /usr/lib/pstore-save/ptestThe ptest output failed to report any results.
Output:
2024-10-03T17:34The ptest output failed to report any results.
Output:
STOP: ptest-runnerThe ptest output failed to report any results.
Output:
TOTAL: 1 FAIL: 1The ptest output failed to report any results.
Output:
```